pyproject.toml

```diff
@@ -327,7 +325,6 @@ lint.ignore = [
     "RET505", "RET506", "RET508",
     "S108", "S113", "S324", "S701",
     "SIM102",
-    "TRY004", "TRY200",
 ]
```

Comment:
You may want to use `select-extend = ["ALL"]` and just focus on maintaining the ignores, e.g.

```toml
[tool.ruff.lint]
extend-select = ["ALL"]
extend-ignore = ["ARG005", "ANN", "D"]

[tool.ruff.lint.per-file-ignores]
"charmcraft/foo.py" = ["INP"]
```